Yoast/wordpress-seo
# Add links to explanation texts

## Summary

This PR can be summarized in the following changelog entry:

* N/A

## Relevant technical choices:

* There is some logic for getting shortlinks in php, which adds some environment info to the links. We chose to expose these shortlinks in the wpseoAdminL10n global, so that we can pass the shortlinks to Javascript. Premium also needs access to these shortlinks (and may require different shortlinks), so we created an integration that can add shortlinks via a filter.

## Test instructions

This PR can be tested by following these steps:

* Verify that in the Focus Keyword Analysis collapsible header, there is an outbound link to the focus-keywords article, with some environment information attached.
* Also, in the developer console, you can verify that the wpseoAdminL10n object contains a shortlinks.focus_keyword_info link.

## Quality assurance


Fixes https://github.com/Yoast/wordpress-seo/issues/10508
